infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -65,6 +66,22 @@ versioned_builds = [
     python_version  = "3.9"
     bundle_libtpu   = "0"
   },
+  {
+    git_tag         = "v2.1.0"
```

Comment:
Who is the 3.9 build with libtpu for? The bundled build is primarily for Kaggle, who uses Python 3.10